application/akka/scala/example/src/test/resources/application-test.conf

```diff
@@ -0,0 +1,3 @@
+include "application"
+
+# default config for tests, we just import the regular conf
```

Comment:
I'd add a blank line at the end of the file.

Reply:
This file was generated automatically by the template of the project, I did no modifications to it. However, one blank line doesn't hurt.